ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -60,3 +59,37 @@ EOF
     echo "${OUTBOUND_TYPE} is not supported yet" || exit 1
   fi
 fi
+
+set +u
+
+if [ -z ${OCP_VERSION} ] && [ -n "${base_ref}" ]; then
```

Comment:
I do not think each workflow in the whole repo always define `OCP_VERSION` or `base_ref`, that totally depends on prow ci job definition.